UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -1240,7 +1246,7 @@ private void AddLayer<T>(LayerType type, string name) where T : Layer.LayerData,
                 }
             }
 
-            int layerDepth = 0;
+            long layerDepth = 0;
```

Comment:
Don't see it right now: why does this need to be a long?

Reply:
To be able to check if new layer depth is more than `int.MaxValue`, because `int.MaxValue + 1` equals `int.MinValue`.

Reply:
Might wanna add a comment for that then